src/Graphics/OpenGLContext/opengl_UnbufferedDrawer.cpp

```diff
@@ -21,63 +24,123 @@ UnbufferedDrawer::UnbufferedDrawer(const GLInfo & _glinfo, CachedVertexAttribArr
 	m_cachedAttribArray->enableVertexAttribArray(rectAttrib::texcoord0, false);
 	m_cachedAttribArray->enableVertexAttribArray(rectAttrib::texcoord1, false);
 
-	m_attribsData.fill(nullptr);
+	int fake;
+	m_attribsData.fill(&fake);
```

Comment:
I know this looks a little weird, but nullptr is a valid value when using VBO's, so if they default to nullptr, the caching fails because it thinks the proper value is already set.

If there is a better way to initialize these to a "useless value that is not NULL" I'm open to it.